types/play-sound/play-sound-tests.ts

```diff
@@ -0,0 +1,24 @@
+import getPlayer = require('play-sound');
```

Comment:
So this module is nowhere using 'getPlayer' convention,
would be better to stick to one used in readme, I'd say:
```ts
import playSound= require('play-sound');
const player = playSound({});
```